<launch>
	<node pkg="kkanbu_tf" type="kkanbu_tf" name="kkanbu_tf" output="screen">
        <param name="ego/init_x" value="719.911269872" />
        <param name="ego/init_y" value="109.211986256" />
        <param name="ego/init_yaw" value="-0.0369807649712" />
        <param name="sensor1/pose_x" value="0.3" />
        <param name="sensor1/pose_y" value="0.0" />
        <param name="sensor1/pose_z" value="1.0" />
    </node>
</launch>